src/test/java/ReverseStringTest.java

```diff
@@ -9,5 +9,14 @@ void verifyTheTestForReversText(){
         Assert.assertEquals(ReverseString.reverseText( "Programming is not easy if you don't know how to swim..."), "...miws ot woh wonk t'nod uoy fi ysae ton si gnimmargorP");
     }
 
+    @Test
+    void verifyTestForReversingStringNull(){
+        Assert.assertNull(ReverseString.reverseText("Programming is not easy if you don't know how to swim..."), "");
```

Comment:
Actually by NULL is meant that you will provide NULL and verify what will be the result.: `reverseText(null)`.

Look at all the aspects of the tests.
You a gave a name to the test case BUT it does not correspond to what it does.